docs/io.cozy.contacts.md

```diff
@@ -41,19 +41,49 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
   - `label?`: {string} A user-provided localized type of instance
   - `primary?`: {boolean} Indicates a preferred-use instance
 - `relationships`: {object} links between documents
+
   - `groups`: {object} groups the contact belongs to
     - `data`: {array} list of groups
       - `_id`: {string} id of the io.cozy.contacts.groups document
-      - `_type`: {string} doctype "io.cozy.contacts.groups" 
-- `metadata`: {object}
-  - `version`: {integer} Used for migrations. Current version is **1**
+      - `_type`: {string} doctype "io.cozy.contacts.groups"
+  - `accounts`: {object}
+    - `data`: {array} list of accounts
+      - `_id`: {string} id of the io.cozy.contacts.accounts document
+      - `_type`: {string} doctype "io.cozy.contacts.accounts"
+
+- `cozyMetaData`: {object}
+
+  - ...`cozyMetadata`: [Document metadata](./README.md#document-metadata)
+  - `sync`: {object}
 
+    - `ded4265b38c54b0683408c76d9ebd`: {object} id of the sourceContactsAccount
+
+      - `konnector`: {string} example : `"google"`
+      - `lastSync`: {date} (example: `"2018-10-19T10:58:37.025688+02:00"`)
+      - `contactsAccountsId`: {string} id of the io.cozy.contacts.accounts object
+      - `id`: {string} id of the remote object
+      - `remoteRev`: {string} latest rev of the remote object
 
 ## `io.cozy.contacts.groups`
 
 Used to group contacts together. A group can obviously have multiple contacts, but contacts can also be part of multiple groups.
 
 - `name`: {string} is the group's public name
-- `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon 
+- `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon
 - `metadata`: {object}
-  - `version`: {integer} Used for migrations. Current version is **1**
+  - `version`: {integer} Used for migrations. Current version is **1**
+
+## `io.cozy.contacts.accounts`
+
+Used to know to which accounts (Google, Apple, etc) a contact is related.
+
+- `id`: {id} id of the io.cozy.contacts.account object
+- `name`: {string} name of the account (example: `"toto@gmail.com"`)
+- `type`: {string} konnector type (example: `"konnector-google"`)
+- `readonly`: {string}
```

Comment:
C'est un booléen je pense, mais je sais pas si `readonly` est très parlant. C'est une option de config qui dit si on peut rattacher des contacts a cette source après coup.

Peut-être `canAddContacts`? `canAttachContacts`?

Reply:
Non. C'est vraiment de savoir si tu peux modifier un contact de cette source (ie Rennes par exemple). un boolean readOnly me va très bien 

Tu parles certainement du `shouldSyncOrphan` @y-lohse ?

Reply:
Non non, je parlais bien de celui la. Dans le paper, on a dit:

> permet d'indiquer qu'une source n'est pas cochable / décochable, par ex. Rennes

Pour moi ca indique bien si un contact peut etre rattaché / détaché de cette source, mais dans tous les cas le contact pourra être modifié.

Reply:
Oh right. `canAttachContacts` semble mieux :)

Reply:
Ceci dit je trouve pas ca terrible non plus. `canAttachContacts` parle juste d'attacher des contacts a cette source, mais ce flag dit aussi si on peut *détacher* un contact.

Peut-être que `editableSource` serait mieux ? Mais ca laisse penser que c'est la source qui est editable ou non, alors que c'est le *rattachement* à cette source qui est éditable ou non…

Reply:
je ne trouve pas mieux. que `canAttachContacts` et `editableSource` (je trouve que "edit" entraîne un peu de confusion, je crois que je préfère `canAttachContacts` et préciser dans la doc que c'est dans les 2 sens)
En tous cas cette discussion m'a permis de mieux comprendre ce dont il s'agit :-)

Reply:
Le `canAttachContacts ` avec un commentaire me va :)

Reply:
`attach` is related to couchdb attachments ? What does attaching a contact mean ? Link ?

Reply:
Attaching a contact = creating a relationship between `io.cozy.contacts` and `io.cozy.contacts.accounts` — so nothing to do with couchdb attachments. But `canRelateContacts` makes no sense, I think. `canAssociateContact` could be another alternative?

Reply:
I personally prefer `canAttachContacts` to `canAssociateContact `and `canRelateContacts`. What do you think of `canLinkContacts` ?